base/compiler/ssair/slot2ssa.jl

```diff
@@ -102,14 +94,12 @@ function fixup_slot!(ir::IRCode, ci::CodeInfo, idx::Int, slot::Int, stmt::UnoptS
     end
     if isa(stmt, SlotNumber)
```

Comment:
I think this is now always true since `stmt::SlotNumber`, which means that `fixup_slot!` always returns `ssa` (even when `ssa === UNDEF_TOKEN`) and the `stmt` argument can be removed.